apps/omg_watcher/lib/web.ex

```diff
@@ -36,6 +36,35 @@ defmodule OMG.Watcher.Web do
       use Phoenix.Controller, namespace: OMG.Watcher.Web, log: :debug
```

Comment:
moduledoc is incorrect 
`use OMG.Watcher.Web,  :view`

Reply:
Agreed, it's OK